lib/puppet/module/task.rb

```diff
@@ -74,16 +74,19 @@ def self.get_file_details(path, mod)
 
     # Find task's required lib files and retrieve paths
     # for both 'files' and 'implementation:files' metadata keys
-    def self.find_files(files, mod)
-      env = mod.environment.respond_to?(:name) ? mod.environment.name : 'production'
```

Comment:
I'm not sure whether this is needed. The info service spec tests fail without it, but that may just be insufficient mocking.